spec/triggers/read.spec.js

```diff
@@ -12,9 +12,9 @@ require('dotenv').config();
 
 describe('SFTP', () => {
   const sftp = new Sftp(bunyan.createLogger({ name: 'dummy' }), {
-    host: process.env.HOSTNAME,
+    host: process.env.SFTP_HOSTNAME,
     port: process.env.PORT,
```

Comment:
Please, replace it with `port: Number(process.env.PORT),` as it has been implemented in `spec-integration/verifyCredentials.spec.js`